Gemfile

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+gem "webrick"
+gem 'public_suffix', '3.0'
+gem 'addressable', '2.8.5'
+gem 'rexml', '3.2.6'
+gem 'kramdown', '2.3.0'
+gem "kramdown-parser-gfm"
+gem 'rouge', '3.19.0'
+gem 'jekyll', '3.9.0'
+gem 'jekyll-archives', '2.1.1'
+gem 'jekyll-sass-converter'
+gem 'jekyll-feed', '0.13.0'
+gem 'jekyll-seo-tag', '2.6.1'
+gem 'jekyll-sitemap', '1.4.0'
+gem 'jekyll-paginate', '1.1.0'
+gem 'liquify', '0.2.7'
+gem 'github-pages', '207'
```

Comment:
Do we really need to specify all of these exact gem versions? It's generally better to specify minimum versions of the key gems (eg, `gem 'github-pages', '>=207'`, and then to let bundler find the latest compatible versions of the rest and save them to the `Gemfile.lock`.

For instance I'm running ruby 3.2.2, which is incompatible with the version of liquid we end up requiring here: https://github.com/github/pages-gem/issues/859
To update this I'd need to specify the liquid version, then find a version of github-pages compatible with that, then find a version of jekyll compatible with that, etc

Otherwise I can just remove all the version specs apart from github-pages and liquid, and just run bundle install/bundle update

Reply:
Another thing I'm not too familiar with. I did have issues with versions so ended up going with the versions suggested in the command line and not wanting to touch it thereafter. 
Incoming message on teams regarding best way edit this